src/halomod/halo_exclusion.py

```diff
@@ -398,8 +479,14 @@ def prob_inner_r_(r, rvir):
 
 
 class NgMatched(DblEllipsoid):
+    r"""
+    A model for double ellipsoid halo exclusion, where a mask is
```

Comment:
A more general comment on the halo exclusion docs: most of these models are taken from Tinker2005, and we should probably put those details into the docstrings (i.e. which equations they come from). Also, the module docstring could contain more mathematical details (could copy a lot of that from my paper).